python/cuml/test/test_coordinate_descent.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import cudf
+import numpy as np
+import pandas as pd
+from cuml import Lasso as cuLasso
+from sklearn.linear_model import Lasso
+from cuml.linear_model import ElasticNet as cuElasticNet
+from sklearn.linear_model import ElasticNet
+from cuml.test.utils import array_equal
```

Comment:
I'm not sure I love the name of this function... numpy already has an array_equal function that checks for exact equality and np.allclose that checks with fuzzy tolerance. Can we just use np.allclose instead of this function?